src/net.h

```diff
@@ -658,8 +664,8 @@ bool AddLocal(const CNetAddr& addr, int nScore = LOCAL_NONE);
 void RemoveLocal(const CService& addr);
 bool SeenLocal(const CService& addr);
 bool IsLocal(const CService& addr);
-bool GetLocal(CService &addr, const CNetAddr *paddrPeer = nullptr);
-CAddress GetLocalAddress(const CNetAddr *paddrPeer, ServiceFlags nLocalServices);
+bool GetLocal(const CService &addr, const CNetAddr *paddrPeer = nullptr);
```

Comment:
This is wrong. `CService` shouldn't be const, and making it const here makes this declaration different from the definition in the cpp file.

In fact, this declaration could just be removed entirely from this header file since the function isn't used outside the translation unit. You could also make it static in the cpp file to remove external linkage.